lib/wanda/operations/state.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Wanda.Operations.State do
+  @moduledoc """
+  State of an operation.
+
+  The state is composed by the next field:
+  - engine: RHAI engine used to run predicate validations
+  - operation_id: Identifier of this operation execution
+  - group_id: Identifier of the group or resource that is under the operation.
+              For example, this can be the identifier of an individual host or a cluster.
```

Comment:
I wonder if we are leaking some Trento concepts into Wanda. What's the meaning of `group_id` in the operation execution domain?

Reply:
I don't think so. The group id merely identifies a group of resources that are grouped together to represent something that must be together. The `group` concept with this naming at least doesn't exist anywhere else.
At the end, the checks and now operation executions are something that are done in a number of targets (a group), and we need an identifier for it.

The docstring is just an example so everyone has an easier time to understand it.